### ISSUE TITLE
Enhance Grunt document

### ISSUE THREAD

**Issue by nhduy1985** [Xceptance/SiteGenesis-Community-TestSuite#85]
Requires 'junit' in section 'XLT Framework for Grunt or Gulp based Execution'

**Comment by nhduy1985** [comment]
it seems that 'junit' is included in 'xlt-4.6.1\lib' but not yet copy to running Grunt

Error message

```
D:\Demandware\Tools\xceptance>grunt
Running "all" task

Running "javaversion" task
Vers= 1.8.0_66

Running "checkPrerequisites" task

Running "compile" task

Running "runall:undefined" (runall) task
java  -Dlog4j.configuration="file:config/js-log4j.properties" -cp "..\xlt-4.6.1\
lib\*:classes:config" org.junit.runner.JUnitCore tests.account.TAccount_CreateAc
count
Error: Could not find or load main class org.junit.runner.JUnitCore
Warning: Command failed: java  -Dlog4j.configuration="file:config/js-log4j.properties" -cp "..\xlt-4.6.1\lib\*:classes:config" org.junit.runner.JUnitCore tests.account.TAccount_CreateAccount
Error: Could not find or load main class org.junit.runner.JUnitCore
 Use --force to continue.

Aborted due to warnings.
```

**Comment by nhduy1985** [comment]
Found out that Windows uses semi-colon ";" instead of ":" for classpath's separator http://docs.oracle.com/javase/6/docs/technotes/tools/solaris/classpath.html

**Comment by occupant23** [comment]
Hi Dustin,

Thank you for pointing out the fix for this one. I will see that I include it within my next update. 
Unfortunately this one can't be fixed with just setting an option.

Thanks,
Marcel

**Comment by occupant23** [comment]
Fixed with pull request #88.
